slice/IceStorm/Metrics.ice

```diff
@@ -17,26 +17,26 @@
 ["swift:module:IceStorm:MX"]
 module IceMX
 {
-    /// Provides information on IceStorm topics.
+    /// Provides information about IceStorm topics.
     class TopicMetrics extends Metrics
     {
-        /// Number of events published on the topic by publishers.
+        /// The number of events published on topics by publishers.
```

Comment:
Looking at the IceStorm code, it does not appear we create a TopicMetrics per-topic ... however, I could be mistaken.

Reply:
I think this depend on the configured metrics views, the per-topic is just the "id" attribute.

https://github.com/zeroc-ice/ice/blob/e67ce93b8da2b79cede68b5db77af1859723194f/cpp/src/IceStorm/InstrumentationI.cpp#L26-L27

Reply:
Fixed.